### ISSUE TITLE
FastEndpoints not working if your library name starts with FastEndpoints

### ISSUE THREAD

**Issue by VadimPestryanin** [FastEndpoints/FastEndpoints#401]
Check EndpointData.cs line 37. For some reason there is strange string comparison which blocks endpoint detection if your library name starts with "FastEndpoints".
This should be documented at least. I've spent more than a hour trying to find out what's wrong and debugging your library.

**Comment by dj-nitehawk** [comment]
yeah it's for speeding up type discovery by excluding some namespaces that shouldn't probably be having any endpoints.

why does your namespace start with "FastEndpoints"?

will add to docs anyway...
thanks!

**Comment by VadimPestryanin** [comment]
why does your namespace start with "FastEndpoints"? cause i've made it like "FastEndpointsPlayground.Web" :)
